web/frontend/src/pages/form/GroupedResult.tsx

```diff
@@ -27,14 +38,26 @@ type GroupedResultProps = {
 // Functional component that displays the result of the votes
 const GroupedResult: FC<GroupedResultProps> = ({ rankResult, selectResult, textResult }) => {
   const { formId } = useParams();
-
-  const { result, configObj } = useForm(formId);
+  const navigate = useNavigate();
+  const { t } = useTranslation();
+  const { loading, result, configObj } = useForm(formId);
   const configuration = useConfigurationOnly(configObj);
 
+  const questionIcons = {
+    [RANK]: <SwitchVerticalIcon />,
```

Comment:
Sweet! I'm wondering, why do you need to use `[RANK]` and can't you use just `RANK` or `RANK.Type` ?

Reply:
I used the [] to convert the value into a string key. RANK.Type does not exist and RANK by itself is considered to be a JSX element and not a string.